fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/dimension/WikiDimensions.java

```diff
@@ -74,9 +78,10 @@ public WikiDimensions() {
      *
      * @return set of dimension configurations
      */
-    public LinkedHashSet<DimensionConfig> getDimensionConfigurationsByApiName(WikiApiDimensionName... dimensionNames) {
+    public LinkedHashSet<DimensionConfig> getDimensionConfigurationsByApiName(
+        WikiApiDimensionConfigInfo... dimensionNames) {
```

Comment:
`) {` should be on the next line down, aligned with the start of the method declaration line